packages/walletkit/src/attestations.ts

```diff
@@ -182,12 +182,12 @@ export function messageContainsAttestationCode(message: string) {
 
 export function extractAttestationCodeFromMessage(message: string) {
```

Comment:
i feel that we are a little inconsistent with throwing vs nulls - wdyt?

Reply:
Agreed. I'm generally in favor of exceptions but nulls were helpful here. Hbu WDYT?